docs/specification.md

```diff
@@ -248,6 +248,18 @@ values = { 1 = "male", 2 = "female", 3 = "non_binary" }
 description = "Sex at Birth"
 ```
 
+Sometimes, you may want to keep the value as-is if no match was found; this
+could be useful when mapping to controlled terminologies, where you want to map
+the value to a known set if found, but keep the value as free text if not. This
+can be done by adding `ignoreMissingKey = true` to the rule:
+
+```ini
+[table.sex_at_birth]
+field = "sex"
+values = { homme = "male", femme = "female" }
+ignoreMissingKey = true
+```
+
```

Comment:
Could you expand this a bit to explain what you'd expect to see if this is applied to data with a missing key? Just to make it a bit clearer

Reply:
thanks, added